bindings/c/include/conv.hh

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <manifold.h>
+#include <meshIO.h>
+#include <public.h>
+#include <sdf.h>
+#include <types.h>
+
+#include <vector>
+
+#include "glm/ext/matrix_float4x3.hpp"
```

Comment:
What is this for?

Reply:
I think they were auto added my my editor, I should be able to remove them.

Reply:
Went through and pruned a bunch of auto-included things. The C++ LSP running in my emacs must be doing it.